parallel-install/pkg/preinstaller/preinstaller.go

```diff
@@ -231,3 +242,21 @@ func findOnlyDirectoriesAmong(input []os.FileInfo) (o []os.FileInfo) {
 
 	return o
 }
+
+func updateLabelIfNecessaryOf(obj *unstructured.Unstructured, label string) {
```

Comment:
The name of this function is a bit misleading, in the end it is just adding a given label to the object.
I would just name it `addLabel()`.